static/js/stream_create.js

```diff
@@ -16,6 +16,27 @@ exports.get_name = function () {
     return created_stream;
 };
 
+var stream_subscription_error = (function () {
+    var self = {};
+
+    self.report_no_subs_to_stream = function () {
+        $("#stream_subscription_error").text(i18n.t("Stream must have at least one subscriber."));
+        $("#stream_subscription_error").show();
+    };
+
+    self.cant_create_stream_without_susbscribing = function () {
+        $("#stream_subscription_error").text(i18n.t("Non realm admin users can't create stream without subscribing."));
```

Comment:
This text is unnecessarily convoluted.  User-facing error strings are worth thinking carefully about and making clear, like your best formal writing. 

This one should be something like "You must be a realm administrator to create a stream without subscribing"

(Sidenote: I guess since it's basically impossible to get here since one can't uncheck the box, do we actually need this error message?)

Reply:
Yes, we have disabled current user checkbox but user can easily enable checkbox from dev tools by removing "disabled" property from checkbox and create stream without subscribing. Which look likes hack, so I add condition to check(even we have disabled checkbox) current user subscribed or not and if not throw an error. Does that make sens? Or should I remove it?